### PR TITLE
Models and validations for micro-Reddit

### PR DESCRIPTION
**Pull request by sjcco**
- Create a model for user
- Create a model for post
- Create a model for comments
- Add validations for each model
- Add an association between models

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
STATUS: Changes Required :red_circle: 

### Hello @sjcco, great job with the project. Your project is working as expected and you have done good work in your model and with validations but you have to improve the following before moving on:
- [x] Add a descriptive readme ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))


_Please implement all the above mentioned requirements and ask for another code review whenver you are ready_

### **If you have any question related to the requirements please add a comment here and ping me on slack @GzimAsani**

# HAPPY CODING :computer: :desktop_computer:

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Carlos Campos Ojeda** 
Good job with the improvements for this project :muscle:  :muscle:
It meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [ ] I guess you were trying to make a list with dashes in the README file. Fix this so your README file looks ok. 



- [ ] The how to set up the project is not complete. Make sure to provide all the necessary instructions.
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

File: README.md [hunk 1]
```diff
@@ -1,24 +1,54 @@
...
+- SQLite3
 
-* Database initialization
+To get a local copy up and running follow these simple example steps.
 
-* How to run the test suite
+### Usage
 
-* Services (job queues, cache servers, search engines, etc.)
+-Clone the repo "https://github.com/sjcco/micro-reddit.git".
```
Comment (whiz25) [inline README.md]: - Write the full command for cloning: `git clone ...`

File: README.md [hunk 2]
```diff
@@ -1,24 +1,54 @@
...
+-In the terminal write '''rails console'''.
+-Start adding users, posts and comments.
```
Comment (whiz25) [inline README.md]: - How does one add a user. Add an example for adding users from the console. Always be explicit in your instructions.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nice work making the requested changes. Let's make some changes to the readme.
I have left comments with suggestions.


### Martin Afani
#### Slack: whiz25

**Review by GzimAsani — APPROVED** [review]
# STATUS: ACCEPTED :green_circle: :1st_place_medal: 

## Hello again @sjcco. Thank you for making all the required changes your project now looks complete and you readme looks professional. You may merge and good luck with the next projects :+1: 

_**Don't forget to submit the project completion form**_

# HAPPY CODING :computer: :desktop_computer:

**Pull request merged**